app/services/rural-payments-portal.js

```diff
@@ -145,6 +145,27 @@ class RuralPaymentsPortal {
 
     return customerResponse.data._data
   }
+
+  async getOrganisationBySBI (sbi) {
+    const cached = cache.get(`/api/organisation/${sbi}`)
```

Comment:
Are we planning to keep this caching as may need to run it past RP?

Reply:
yes, some graphql queries can call this function many times in the same request. so this prevents us doing the same request many times. We could reduce the time its cached for?

Reply:
The `RESTDataSource` class solves the timing issue as its cache is just a per-request cache to deduplicate calls to the send endpoint that might occur in a single query